manifoldDeps.cmake

```diff
@@ -3,6 +3,7 @@ include(GNUInstallDirs)
 find_package(PkgConfig QUIET)
 
 if(MANIFOLD_PAR STREQUAL "TBB")
+    find_package(Threads REQUIRED)
```

Comment:
What's this? Why do we need this dependency now and didn't before?

Reply:
This is pthreads. Tbb always require pthreads, but for some reason with mxe it somehow does not detect pthread and we need to add this find package call. OpenSCAD does not have this call but it somehow works, I don't have much idea why we were getting errors regarding this...